src/schema/basic_slots.yaml

```diff
@@ -34,6 +35,17 @@ default_range: string
 
 slots:
 
+  processing_institution_workflow_metadata:
+    range: string
+    description: >- 
+      Information about how workflow results were generated when the processing is done by an external organziation (e.g., JGI) such
+      as software tool name and version or pipeline name and version.
+    mappings:
+      - EDAM:0949
```

Comment:
I'm not used to this style of EDAM identifiers that doesn't have text before the numbers, like these prefix expansions demonstrate

  edam.data: "http://edamontology.org/data_" # for doi_value and DataCategoryEnum only
  edam.format: "http://edamontology.org/format_" # for 'Configuration toml' PV only only
  
  
https://bioregistry.io/registry/edam


It's great the a mapping has been provided here, along with an expansion. But we should always think twice about adding mappings that require a new prefix expansion, especially if it is